### ISSUE TITLE
Generate a large labelled dataset of cropped nutrition tables from Open Food Facts data

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-ai#17]
Generate a large labelled dataset of cropped nutrition tables from Open Food Facts data
This will enable autocropping systems.

**Comment by Asteur** [comment]
@teolemon still actual?

**Comment by teolemon** [comment]
We have moved forward on this, so I believed the dataset part is fixed @Asteur 
Would you be interested in helping with AI tasks ? We have a weekly call on Nutrition extraction.

**Comment by Asteur** [comment]
Yes, AI or in python @teolemon In slack?

**Comment by teolemon** [comment]
yup, there's an important  issue in Python that would have quite an impact waiting for you in the #robotoff channel @ https://slack.openfoodfacts.org :-)

**Comment by Asteur** [comment]
Do you mean this one https://github.com/openfoodfacts/robotoff/issues/260? <https://github.com/openfoodfacts/robotoff/issues/260?> Let me take a look, what’s going on in #robotoff
